src/Xamarin.Forms.PancakeView.Multi/Platforms/Tizen/PancakeViewRenderer.cs

```diff
@@ -0,0 +1,195 @@
+﻿using System;
+using System.ComponentModel;
+using System.Linq;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Tizen;
+using Xamarin.Forms.Platform.Tizen.Native;
+using Xamarin.Forms.PancakeView.Tizen;
+using Controls = Xamarin.Forms.PancakeView;
+using SkiaSharp;
+using SkiaSharp.Views.Tizen;
+
+[assembly: ExportRenderer(typeof(Controls.PancakeView), typeof(PancakeViewRenderer))]
+namespace Xamarin.Forms.PancakeView.Tizen
+{
+    public class PancakeViewRenderer : LayoutRenderer
+    {
+        SKCanvasView _skCanvasView;
+        PancakeView PancakeViewElement => Element as PancakeView;
+
+        public PancakeViewRenderer() : base()
+        {
+        }
+
+        /// <summary>
+        /// This method ensures that we don't get stripped out by the linker.
+        /// </summary>
+        public static void Init()
+        {
+#pragma warning disable 0219
+            var ignore1 = typeof(PancakeViewRenderer);
+            var ignore2 = typeof(PancakeView);
+#pragma warning restore 0219
+        }
+
+        protected override void OnElementChanged(ElementChangedEventArgs<Layout> e)
+        {
+            if (Control == null)
+            {
+                SetNativeControl(new Canvas(Forms.NativeParent));
+                _skCanvasView = new SKCanvasView(Forms.NativeParent);
+                _skCanvasView.PaintSurface += OnPaintSurface;
+                _skCanvasView.Show();
+                Control.Children.Add(_skCanvasView);
+                Control.LayoutUpdated += OnLayoutUpdated;
+            }
+            base.OnElementChanged(e);
+        }
+
+        protected override void Dispose(bool disposing)
+        {
+            base.Dispose(disposing);
+            if (disposing)
+            {
+                if (Control != null)
+                {
+                    Control.LayoutUpdated -= OnLayoutUpdated;
+                }
+                if (_skCanvasView != null)
+                {
+                    _skCanvasView.PaintSurface -= OnPaintSurface;
+                    _skCanvasView.Unrealize();
+                    _skCanvasView = null;
+                }
+            }
+        }
+
+        protected override void UpdateBackgroundColor(bool initialize)
+        {
+        }

```

Comment:
?

Reply:
I intended to remove the base call of updating `BackgroundColor`, but found a better way. Fixed.